src/elementPlugins/types.ts

```diff
@@ -22,6 +22,16 @@ interface ApplicationViewProps extends ApplicationViewWrapperProps {
   onUpdate: OnUpdateApplicationView
 }
 
+interface SummaryViewProps {
+  parameters: any
+  value: ResponseFull | null
+}
+
+interface SummaryViewWrapperProps {
+  element: any
+  value: ResponseFull | null
+}
```

Comment:
Those two types are very similar now, would they require different input? If that's the case can it be more finely defined?

Reply:
Okay, have made the second one more specific. For the first I've created a new type called "BasicObject", which is essentially an object of any shape, same as in back-end. Do you think this is necessary, or should we leave "parameters" as `any`. (Parameters is an object but properties could be anything)

Reply:
Yes. That's better, thank you. I think it would be good at some point to track all possible properties (that we allow plugins to have - maybe not the keys but the types of the objects' properties). But not for now.